### PR TITLE
Translation

### PR DESCRIPTION
**Pull request by Roj**
I added the new file and folder, which includes the translation.
Fixed the error_not_cached element (line 303) and modified line 470 to include the placeholder $MOSTLY_HARMLESS$, since you used it before and after but apparently you forgot to do so in that element :P

Please take in mind that I am not, by any means, a professional translator or anything like one, I'm just a native speaker of Spanish. I modified the description attributes of each element so others can modify my translation in an easier way.

### CODE REVIEW

File: _locales/en_US/messages.json [hunk 1]
```diff
@@ -603,7 +607,7 @@
 		"message": "Excluded regex"
 	},
 	"excluded_regex_description": {
-		"message": "Put regular expressions, one per line, in the above text box. Any page on the listed domains will not activate $MOSTLY_HARMLESS$.",
+		"message": "Put regular expressions, one per line, in the above text box. Any page that matches these regular expressions will not activate $MOSTLY_HARMLESS$.",
```
Comment (Roj) [inline _locales/en_US/messages.json:610]: I think this suits better.

**Comment by Kerrick** [comment]
I think I merged that right... this is my first pull request. :D

So, what would you like your icon to be on [the credits page](http://kerrick.github.com/Mostly-Harmless/#credits)?

**Comment by Roj** [comment]
Ah, uhm, [this one?](http://i.imgur.com/Np25X.png)

**Comment by Kerrick** [comment]
Done! http://kerrick.github.com/Mostly-Harmless/#credits

**Pull request merged**